README.md

```diff
@@ -1,18 +1,38 @@
-# Governance
+# ESLint-Community Governance
 
-Outlines the governance of [@eslint-community](https://github.com/eslint-community) and fascilitates proposals and nominations
+## Purpose
 
-## Goal
+To have a place where community members can help ensure that widely depended upon ESLint-related packages live and never fall out of maintenance.
 
-As you can read in the
-["`eslint-community` GitHub organization" RFC](https://github.com/eslint/rfcs/tree/main/designs/2022-community-eslint-org),
-the goal of this organization is to have a place where community members can
-help ensure widely depended upon ESLint-related packages live and never fall out
-of maintenance.
+## Governance
+
+See [GOVERNANCE.md](./GOVERNANCE.md)
 
 ## The Community Core team
 
 * [`@aladdin-add`](https://github.com/aladdin-add)
 * [`@MichaelDeBoey`](https://github.com/MichaelDeBoey)
 * [`@ota-meshi`](https://github.com/ota-meshi)
 * [`@voxpelli`](https://github.com/voxpelli)
+
+
+## Projects
+
+### Primary projects
+
+* [eslint-plugin-es-x](https://github.com/eslint-community/eslint-plugin-es-x) (Collaborators: [@brettz9](https://github.com/brettz9))
```

Comment:
I haven't worked with this particular project, though I was added subsequently to the `eslint-plugin-promise` project.

Reply:
Updated 🙏